### PR TITLE
wicked: add devel tools module to AutoYaST profile for SLE 15

### PR DESCRIPTION
**Pull request by asmorodskyi**
To be able to install wicked from sources we need to install
pattern Basis-Devel . Current AutoYaST profile do not have all
needed modules enabled

### CODE REVIEW

**Comment by asmorodskyi** [comment]
VR:  http://dante.arch.suse.de/tests/271

File: tests/wicked/before_test.pm [hunk 1]
```diff
@@ -64,8 +64,7 @@ sub run {
     } else {
         # Common SUT Configuration
         if (get_var('WICKED_SOURCES')) {
-            zypper_call('--quiet in -t pattern Basis-Devel');
```
Comment (okurz) [inline tests/wicked/before_test.pm]: why not install the pattern anymore? I did not see it covered in the autoyast profile

Comment (asmorodskyi) [reply tests/wicked/before_test.pm]: I failed to find pattern in SLE 15 SP2 and while trying to solve this somehow I figure out that actually I don't need a pattern but only two packages make and gcc . And even if I will find pattern in SLE 15 SP2 ( what I plan to do anyway ) I still prefer to keep it this way ( I checked that it working for SLE 12 SP5 too )

**Comment by schlad** [comment]
Could you please correct mistakes in git commit message please?
Notably: " . "
and "Current AutoYaST profile do not"; I reckon it should be 'does'

**Comment by asmorodskyi** [comment]
done

File: tests/wicked/before_test.pm (hunk 1 above)
Comment (schlad) [reply tests/wicked/before_test.pm]: Personally I prefer having all required packages listed explicitly as this is not obscuring potential problems in the future.
If we want to test if given pattern installs expected packages, we could have slim and inexpensive test for that (or we perhaps have that already for this pattern).

**Comment by schlad** [comment]
@okurz I've merged it as you are apparently on holidays today and Anton would like to get this done. Issue which you asked about seems like a valid question but we could easily come back to this once you are back.

**Comment by okurz** [comment]
all good :)

**Pull request merged**